script/Ciel/Day 13/13_01_01_CIEL13_4.txt

```diff
@@ -504,98 +538,106 @@ That alone is the undeniable truth.
 }
 [sha:4828faac957540f24fc1dcefbe8e6364a14ac37e]{
 -- Page 340, Offset 34199.
 -- 視界が滲んでいく。
-My vision is getting blurry.
+My vision is starting to blur.
 }
 [sha:85882482b1a30e16bb4cea7e964b874ee0614d7e]{
 -- Page 340, Offset 34200.
 -- 頭上には煌々と輝く白い月。
-The dazzling white moon shines high in the sky.
+Far above me, the moon gleams a radiant white.
 }
 [sha:156b017468b7e11872dc6bbde9df130027d77972]{
 -- Page 340, Offset 34201.
 -- 灰色の瓦礫で埋まった静かの海。
-A tranquil sea, surrounded by grey rubble.
+// Assuming this is filled -> comprised of for 埋まる
+Below, a tranquil ocean of grey rubble.
 }
 [sha:e4d3106bc7695e30a54f06e95c41dc6e719eca99]{
 -- Page 340, Offset 34202.
 -- ゆらゆらと揺らいでいる。
-The moon is swaying and wavering.
+Everything is swaying and flickering.
 }
 [sha:4c11e69126bba185ae381bea6fb514f2cc1a84fb]{
 -- Page 340, Offset 34203.
 -- 何もかも偽りで、近づけば消えてしまう幻。
-With everything veiled in lies, it's an illusion that vanishes when you try to approach it.
+It's all just lies; an illusion that vanishes when you try and get too close.
 }
 [sha:99593aeee6019fa1d1feff11f3c09ffd0feaa731]{
 -- Page 340, Offset 34204.
 -- それは一つの、掴みようのない蜃気楼のようだった。
-It's just like a lone mirage that remains out of my reach.
+A mirage that will forever remain out of reach.
 }
 [sha:e8bca9edde6d470caab5ca5e784aafb0a5f76c3b]{
 -- Page 341, Offset 34205.
 -- かつん、かつん。
-%{i}Step%{/i}. %{i}Step%{/i}.
+%{i}Tap%{/i}. %{i}Tap%{/i}.
 }
 [sha:5957b026b133dac97f93c39e4cb8857aa4ea2e58]{
 -- Page 341, Offset 34206.
 -- 海の底で乾いた足音が響きだした。
 // fuck yeah nasu 'dry' is definitely a very good descriptor for footsteps
-The sound of dry footsteps echoes throughout the bottom of the sea.
+// he uses it _constantly_. I get what he means but yeah
+The rasping sound of footsteps echoes across the ocean floor.
 }
 [sha:d252f673316cca02fa057d487903f1f852b4bd9b]{
 -- Page 341, Offset 34207.
 -- 月光を背に、荒涼とした大地に伸びる断罪者の影。
-As the moon shines behind her, the adjudicator's shadow is cast upon the desolate ground.
+// Sorry weblio, I don't think 断罪者 means 'Tetragrammaton Labyrinth'
+// https://ejje.weblio.jp/content/%E6%96%AD%E7%BD%AA%E8%80%85
+// Word doesn't seem to be used elsewhere, dunno if it's supposed to be a term or not
+With the moon at her back,
+ the adjudicator casts a long shadow across the desolate ground.
 }
 [sha:f3d2d5caf159731f824f7715a918aa7d3e2baf51]{
 -- Page 342, Offset 34208.
 -- ……間違いなくこれでおしまい。
-...There's no doubt that this is the end.
+There's no doubt that this is the end...
 }
 [sha:dc6f2ecc97d847a605670bb6e18a756e5430e44a]{
 -- Page 342, Offset 34209.
 -- ……彼女がもうすぐやってくる。
-...She'll arrive here at any moment now.
+She'll reach me any moment now.
 }
 [sha:74a7b0dd9e2bdadf2b0c3203c008de6bd9ff0e39]{
 -- Page 342, Offset 34210.
 -- 色々あったけど結論は出せた。
-A lot has happened along the way, but now we've finally reached the conclusion.
+// I assume it's the situation that's been concluded, rather than him reaching a conclusion about something?
+It's been a wild ride, but we've reached the end of the line.
```

Comment:
I'd say it's the latter? It's a transitive verb being used, after all. The conclusion being that he doesn't want to die.

>It's been a wild ride, but I've reached my conclusion.